bin/cron4rucio_ds_mongo.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
+set -e
+##H Can only run in K8s, you may modify to run in local by arranging env vars
+##H
+##H cron4rucio_ds_mongo.sh
+##H    Cron job of rucio_ds_mongo.py which runs Spark job to get Rucio datasets and writes to HDFS directory.
+##H    After writing data to HDFS directory, it copies HDFS files to LOCAL directory as a single file.
+##H
+##H Usage:
+##H    cron4rucio_ds_mongo.sh --keytab <value> --mongouser <value> --mongopass <value>
+##H
+##H Example :
+##H    cron4rucio_ds_mongo.sh \
+##H        --keytab ./keytab \
+##H        --mongouser $MONGO_ROOT_USERNAME \
+##H        --mongopass $MONGO_ROOT_PASSWORD
+##H
+##H Arguments with values:
+##H   - keytab       : Kerberos auth file to connect Spark Analytix cluster (cmsmonit)
+##H   - mongouser    : MongoDB user which has write access to required MongoDB database/collection
+##H   - mongopass    : MongoDB user password
+##H
+##H Requirements:
+##H   - Kubernetes service name should be "cmsmon-rucio-ds-mongo". If not please modify Spark ports envs accordingly.
+##H   - "mongoimport" CLI tool executable path should be set in $PATH.
+##H
+##H Environment Variable Requirements
+##H   - MONGO_HOST
+##H   - MONGO_PORT
+##H   - MONGO_WRITE_DB                            : stores the spark results
+##H   - MY_NODE_NAME                              : spark.driver.host
+##H   - CMSMON_RUCIO_DS_MONGO_SERVICE_PORT_PORT_0 : comes from K8s service NodePort
+##H   - CMSMON_RUCIO_DS_MONGO_SERVICE_PORT_PORT_1 : comes from K8s service NodePort
+##H   - WDIR                                      : working directory
+##H
+##H References:
+##H   - CMSSpark/bin/cron4rucio_datasets_daily_stats.sh
+##H
+
+trap 'onFailExit' ERR
+onFailExit() {
+    echo "$(date --rfc-3339=seconds)" "[ERROR] Finished with error!"
+    exit 1
+}
+usage() {
+    perl -ne '/^##H/ && do { s/^##H ?//; print }' <"$0"
```

Comment:
please get rid off perl stuff which creates extra dependencies. You can do the same with UNIX sed command and do not need perl per-se.

Reply:
Thanks @vkuznet , changed with sed/grep